amqp-utils/src/main/java/io/enmasse/amqp/Artemis.java

```diff
@@ -46,15 +46,17 @@
     private final Vertx vertx;
     private final ProtonConnection connection;
     private final ProtonSender sender;
+    private final ProtonReceiver receiver;
```

Comment:
I see that we now pass this in to the constructor, and initialize there... but is it actually being used somewhere, or is this just something to stop things being collected by GC or something...?

Reply:
It was to be able to issue credits manually, which I removed. But on the other hand it felt natural to leave it in just in case :)